### PR TITLE
Try to update ruby version to 2.4.0

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: test/hash_test.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 require 'hanami/utils/hash'
 
 describe Hanami::Utils::Hash do
+  CURRENT_RUBY_VERSION = defined?(TRAVIS_RUBY_VERSION) ? TRAVIS_RUBY_VERSION : RUBY_VERSION
```
Comment (jodosha) [inline test/hash_test.rb]: @davydovanton Is this new constant defined because Travis CI doesn't support 2.4 yet? Can we get rid of this?

Comment (jodosha) [inline test/hash_test.rb]: Do you mean they ship a patched version of Ruby where `RUBY_VERSION` isn't available?

**Comment by davydovanton** [comment]
@jodosha sorry, it was my mistake. I mixed up the rubocop error to the travis. Fixed now 👍

**Comment by jodosha** [comment]
@davydovanton Assign it to yourself and merge it! :wi

**Comment by davydovanton** [comment]
thanks for review @jodosha 👍

**Pull request merged**